src/spotify/spotify-config.ts

```diff
@@ -5,36 +5,14 @@ import type { AccessToken } from "@spotify/web-api-ts-sdk";
 // TODO Schema decode
 const accessToken: AccessToken = AccessTokenJson as unknown as AccessToken;
 
-export type ISpotifyConfig = Readonly<{
-  accessToken: AccessToken;
-  clientId: string;
-  clientSecret: Secret.Secret;
-  port: number;
-  redirectServerPath: string;
-}>;
-
-const make = Effect.gen(function* (_) {
-  const clientId = yield* Config.string("SPOTIFY_CLIENT_ID");
-  const clientSecret = yield* Config.secret("SPOTIFY_CLIENT_SECRET");
-  const port = yield* Config.number("REDIRECT_SERVER_PORT").pipe(
+export const SpotifyConfig = Config.all({
```

Comment:
Leaving config as a `Config` data type means one less service to wrangle. But this is personal preference really.